lib/constants.ts

```diff
@@ -58,8 +63,18 @@ export const DEFAULT_DEVELOPMENT_PLAN: DevelopmentPlan = {
   startDate: "",
 };
 
-export const DEFAULT_TIMETABLE_EVENTS = developmentPlanTimetableEvents.map(
-  ({ key }) => ({
+const eventsToExclude = new Set<TimetableEventKey>([
+  "timetable-updated",
+  "plan-paused",
+  "plan-withdrawn",
+  "plan-found-sound",
+  "plan-found-unsound",
+  "plan-not-adopted",
+]);
+
+export const DEFAULT_TIMETABLE_EVENTS = developmentPlanTimetableEvents
+  .filter(({ key }) => !eventsToExclude.has(key))
```

Comment:
This might be a temporary solution, we can see if there's a smoother way to do this when we handle the refactoring after algining with the visualisations in the design (mentioned in the PR description).